### DISCUSSION TITLE
New Train

### DISCUSSION THREAD

**Discussion by ghost** [ria-com/nomeroff-net#189] [Show and tell]
Прочитала статью на хабре про трейнинг, как поняла сейчас перешли на торч. Есть ли новые обновления которые  нужно учесть, лайфхаки как трейнить лучше?)

**Comment by ApelSYN** [comment]
Да мы существенно ускорили детекцию и перешли на pytorch. В архитектурном плане изминений немного, смотрите ноутбуки с примерами тренировки, если что-то будет непонятно задавайте вопросы.

**Reply by ghost** [reply]
После тренировки ocr-train нужно ли изменять путь модели или модель сама обновляется?

**Reply by ApelSYN** [reply]
> После тренировки ocr-train нужно ли изменять путь модели или модель сама обновляется?

Не совсем понял вопрос, новые модели создается в папке ./models/ , те что скачиваются автоматически размещаются в папке ./data/models/TextDetector/<ocr_name>/
